frontend/src/libs/components/toggle/toggle.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "src/assets/css/styles.scss";
+
+.toggle {
+  --size: 20px;
+
+  width: calc(var(--size) * 2);
+  min-width: calc(var(--size) * 2);
+  height: var(--size);
+  border: calc(var(--size) / 10) solid var(--color);
+  border-radius: calc(var(--size) / 2);
+}
+
+.toggle::after {
+  --radius: 6px;
```

Comment:
you don't need to add static variables in classnames, and then reuse in calc. there is no sense